browser/plugins/three_clickable_object.plugin.js

```diff
@@ -41,11 +41,16 @@
 
 	ThreeClickableObject.prototype.on_click = function() {
 		this.setClickedOnNextUpdate = true
```

Comment:
also, please change the description of the plugin to note that it requires using an 'On Runtime Event'